src/input/Mouse.cc

```diff
@@ -212,23 +224,39 @@ void Mouse::write(uint8_t value, EmuTime::param time)
 		// uses, but 1.5ms is also the timeout value that is used for
 		// JoyMega, so it seems like a reasonable value.
 		if ((time - lastTime) > EmuDuration::usec(1500)) {
-			phase = PHASE_YLOW;
+			// Timeout to YLOW2 so that MSX software bypassing the BIOS
+			// and potentially not performing the alternate cycle
+			// (which normally is used for trackball detection)
+			// still gets continuous delta updates in each scan round
+			phase = PHASE_YLOW2;
 		}
 		lastTime = time;
 
 		switch (phase) {
-		case PHASE_XHIGH:
-			if ((value & STROBE) == 0) phase = PHASE_XLOW;
+		case PHASE_XHIGH1:
+			if ((value & STROBE) == 0) phase = PHASE_XLOW1;
```

Comment:
Cases `PHASE_XHIGH1`, `PHASE_XHIGH2`, `PHASE_YHIGH1` and `PHASE_YHIGH2` basically all do the same thing. At least if we rewrite it like:
```
case PHASE_XHIGH1: case PHASE_XHIGH2:
case PHASE_YHIGH1: case PHASE_YHIGH2:
    if ((value & STROBE) == 0) ++phase;
    break;
```
And similar for `PHASE_XLOW1` and `PHASE_XLOW2`. (In the original code it wasn't worth merging the phases, but now that we've doubled the number of phases it makes more sense).

But `PHASE_YLOW1` and `PHASE_YLOW2` are really different (that was the point of introducing the 2nd set of phases):
```
case PHASE_YLOW1:
    if ((value & STROBE) != 0) {
        phase = PHASE_XHIGH2;
        xRel = yRel = 0;
    }
    break;
case PHASE_YLOW2:
    if ((value & STROBE) != 0) {
        phase = PHASE_XHIGH1;
        ... same as in your patch ...
    }
    break;
```
Notice how `case PHASE_YLOW1` prepares the result that will be returned in `read()` for the 2nd set of phases. Also notice how to difference in behavior between `PHASE_YLOW1` and `PHASE_YLOW2` is now very close together in the code.